Add zip files with password encryption

jeffhandley opened the pull request:
In preparation for a PR that implements dotnet/runtime#68897, adding an `ZipArchiveEntry.IsEncrypted` property, this PR adds 3 zip files for testing.

* `encrypted_entries_weak,.zip` includes 2 entries with "weak" (ZipCrypto) encryption and 2 entries that are unencrypted
  * Password of encrypted entries is "encrypted_entries_weak"
* `encrypted_entries_strong.zip` includes 2 entries with "strong" (AES-256) encryption and 2 entries that are unencrypted
  * Password of encrypted entries is "encrypted_entries_strong"
* `encrypted_entries_mixed.zip` includes 1 entry with "weak", 1 entry with "strong", and 2 entries with no encryption
  * Password of encrypted entries is "encrypted_entries_mixed"

For each encrypted entry, there is a comment on the entry indicating what the password for that entry is.

Jozkee commented:
1. I see the entries are blank, which is OK for `ZipArchiveEntry.IsEncrypted`, but if the files were not empty, we could re-use the assets for the tests of the rest of the Zip entryption APIs (check the contents are correctly preserved/not garbled).
2. `encrypted_entries_mixed.zip` doesn't have the password in the Comment.
![image](https://user-images.githubusercontent.com/16040868/170092118-77a4bb22-2c1b-4e00-91d5-8ac7fdbfe2f2.png)

jeffhandley commented:
@Jozkee Thanks for the feedback on this. I updated the zips to:

1. Put contents into each file (simply `Filename: <filename-without-extension>`)
2. Rename the "strong" file to "aes256" in case we want to extend the coverage later
3. Ensure all encrypted files have comments with their passwords (and changed the "mixed" passwords to be different between encrypted entries)

jeffhandley commented:
Here's the branch where the feature is implemented and the tests can be seen:
https://github.com/dotnet/runtime/compare/main...jeffhandley:jeffhandley/zippassword?expand=1
